backend/src/services/crud.service.ts

```diff
@@ -16,7 +16,9 @@ export abstract class BaseService<T, C, U> {
    *
```

Comment:
can you also rename the class from BaseService to CRUDService

Reply:
Done.